packages/ndla-ui/src/Breadcrumb/BreadcrumbBlock.tsx

```diff
@@ -6,26 +6,38 @@
  *
  */
 
-import React, { useRef, useLayoutEffect } from 'react';
-import PropTypes from 'prop-types';
-import BEMHelper from 'react-bem-helper';
-import { useComponentSize } from '@ndla/hooks';
+import React, { useRef } from 'react';
+import BEMHelper, { ReturnObject } from 'react-bem-helper';
+import { useComponentSize, useIsomorphicLayoutEffect } from '@ndla/hooks';
 import BreadcrumbItem from './BreadcrumbItem';
 
-const classes = BEMHelper({
+const classes: BEMHelper<ReturnObject> = BEMHelper({
   name: 'breadcrumb-block',
   prefix: 'c-',
 });
 
-const BreadcrumbBlock = ({ children, items }) => {
-  const olRef = useRef(null);
-  const containerRef = useRef(null);
+interface BreadcrumbItemI {
+  to: string;
+  name: string;
+}
```

Comment:
Samme interface er definert to plasser. Kunne dei vore slått sammen?

Reply:
Japp kan eksportere den fra `Breadcrumb.tsx` :+1: